dynamo/plot/state_graph.py

```diff
@@ -105,7 +105,13 @@ def state_graph(
     aggregate = group
 
     points = adata.obsm["X_" + basis][:, [x, y]]
-    groups, uniq_grp = adata.obs[group], list(adata.obs[group].unique())
+    unique_group_obs = adata.obs[group].unique()
+    if type(adata.obs[group]) is np.ndarray:
+        groups, uniq_grp = adata.obs[group], unique_group_obs.tolist()
```

Comment:
@dummyindex I think the original one is enough for the cases you are having here.

Reply:
adata.obs[group] is always a pd.series

Reply:
You are right about it. My point is unique_group_obs is not always pd.Series, which sometimes causes problems. I will update this PR with the correct version fix. Sorry somehow this commit is incorrect due to my previous rebase&merge locally.
![image](https://user-images.githubusercontent.com/12120524/112671821-a2f66b00-8e9d-11eb-95b3-9d3e97f3214a.png)

Reply:
The reason for separating cases instead of using list() is that tolist() and to_list may be more efficient because they are implemented by numpy/pandas developers. If you think list() is enough and code compactness is more important, I can revert this change.

Reply:
this should good to be. the conversion speed is not a huge concern for small datasets but maybe relevant when we have 1 million + cells